### DISCUSSION TITLE
Action doesn't work if one PR contains more commits

### DISCUSSION THREAD

**Discussion by acearth** [GsActions/commit-message-checker#62] [General]
Thanks to the author, we can use this action to check commit message following the proper format. 

But I found the action failed to check following situations:
(1) the PR contains 2 or more commits;
(2) at least one commit in the PR following the proper format;
(3) at least one commit in the PR broken the rule of proper format.
In this case,  the check still treat the situation as proper one then check pass. but in the controversary, the action check shall be failed due to "at least one commit broken the rule of format".

Can you let me know, how to enable the action to check on every commit of one PR. Or, is it possible to improve in next version?

**Comment by gilbertsoft** [comment]
@acearth Please check the `checkAllCommitMessages` option see https://github.com/GsActions/commit-message-checker#configuration

**Comment by InPermutation** [comment]
The example in README.md refers to

```
gsactions/commit-message-checker@v1
```

but that release doesn't contain the `checkAllCommitMessages`.

**Reply by gilbertsoft** [reply]
Should be fixed with the today's release, please check.

**Comment by JobaDiniz** [comment]
Does the `checkAllCommitMessages` check **only** PR commits? Or does it check **all** the commits in the repository?
Because in my case, the team started to adopt standard commit messages, but there are a lot of commits in the history that does not follow the pattern.

**Reply by gilbertsoft** [reply]
It's about the commits in a PR or a push. So if you push multiple commits each should be checked properly.
